src/components/scenes/ManageTokensScene.tsx

```diff
@@ -97,6 +111,35 @@ function ManageTokensSceneComponent(props: Props) {
     })
   }, [allTokens, searchValue, sortedTokenIds])
 
+  // Split the list of tokens based on if there were auto-detected tokens given
+  const autoDetectedTokenIds = React.useMemo(
+    () => (newTokenIds ? filteredTokenIds.filter(filteredTokenId => newTokenIds.includes(filteredTokenId)) : []),
+    [filteredTokenIds, newTokenIds]
+  )
+
+  const extraData = React.useMemo(() => ({ allTokens, enabledTokenSet, customTokens }), [allTokens, enabledTokenSet, customTokens])
```

Comment:
If you switch `renderRow` to a `useCallback`, ESLint will help you keep this list up-to-date and correct. You won't need to pass the `extraData` param anymore, because the callback itself will change (`!==`) when its dependencies change.